vsphere/resource_vsphere_host.go

```diff
@@ -91,6 +91,13 @@ func resourceVsphereHost() *schema.Resource {
 				Optional:    true,
 				Description: "Set the host's lockdown status. Default is disabled. Valid options are 'disabled', 'normal', 'strict'",
 				Default:     "disabled",
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
```

Comment:
I believe `validate.StringInSlice` will do the same for you here, and then you don't need to write a new function.